### PR TITLE
Enhancement: Update Metadata

### PR DESCRIPTION
**Pull request by brolewis**
Get better metadata into the project before we upload to PyPI

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -4,6 +4,10 @@ version = "2.1.0"
 description = "A collection of tools for interactive with GitHub"
```
Comment (ecedmondson) [inline pyproject.toml]: not a part of this pr, but... "for interactive" doesn't sound exactly right

**Review by sdutton-personal — APPROVED** [review]
LGTM

**Comment by brolewis** [comment]
Slapping `do not merge` on this one until I can successfully upload to `test.pypi.org` and verify the `README.rst` works OK.

File: pyproject.toml (hunk 1 above)
Comment (brolewis) [reply pyproject.toml]: _I'm certain I don't know what you mean_

**Review by bradsbrown — APPROVED** [review]
LGTM

**Review by ecedmondson — APPROVED** [review]

**Pull request merged**